src/gz/z64.h

```diff
@@ -770,7 +770,9 @@ typedef struct
   char              unk_0x13D6[0x000A];       /* 0x13D6 */
   int8_t            seq_index;                /* 0x13E0 */
   int8_t            night_sfx;                /* 0x13E1 */
-  char              unk_0x13E2[0x0018];       /* 0x13E2 */
+  char              unk_0x13E2[0x0006];       /* 0x13E2 */
+  uint16_t          hud_flag;                 /* 0x13E8 */
+  char              unk_0x13EB[0x10];         /* 0x13EB */
```

Comment:
This member is mislabeled, the address in the name and comment should be `0x13EA`.

Reply:
Resolved